rero_ils/modules/items/api_views.py

```diff
@@ -117,20 +117,20 @@ def librarian_request(item, data):
 
 def prior_checkout_actions(item, data):
     """Actions executed prior to a checkout."""
-    if data.get('loan_pid'):
-        loan = Loan.get_record_by_pid(data.get('loan_pid'))
+    if data.get('pid'):
+        loan = Loan.get_record_by_pid(data.get('pid'))
         if (
             loan.get('state') == 'ITEM_IN_TRANSIT_FOR_PICKUP' and
             loan.get('patron_pid') == data.get('patron_pid')
         ):
             item.receive(**data)
         if loan.get('state') == 'ITEM_IN_TRANSIT_TO_HOUSE':
-            item.cancel_loan(loan_pid=loan.get('loan_pid'))
-            del data['loan_pid']
+            item.cancel_loan(pid=loan.get('pid'))
```

Comment:
loan.pid ?

Reply:
not an invenio_circulation loan class, does not have the pid shortcut